exist-core/src/main/java/org/exist/xmldb/XmldbURI.java

```diff
@@ -479,7 +414,7 @@ protected void recomputeURI() throws URISyntaxException {
     protected void safeRecomputeURI() {
         try {
             recomputeURI();
-        } catch (final URISyntaxException e) {
+        } catch (final URISyntaxException ignored) {
```

Comment:
Why `ignored` when the common syntax is `e`?

Reply:
Because the exception is ignored here, I do like to make it explicit to the reader. Also IntelliJ suggested this change.

Reply:
Okay, I have no strong feeling either way.

Reply:
@line-o I would at least log the exception (even in debug)